ext/js/dom/selector-observer.js

```diff
@@ -170,7 +170,7 @@ export class SelectorObserver {
 
         if (
             this._onChildrenUpdated !== null &&
-            (addedNodes.length !== 0 || addedNodes.length !== 0)
+            (removedNodes.length > 0 || addedNodes.length > 0)
```

Comment:
I think this was a bug, as it was previously checking addedNodes to be not empty twice.